loadtestconfig.default.json

```diff
@@ -59,6 +60,7 @@
         "ChannelLinkChance": 0.02,
         "UploadImageChance": 0.01,
         "LinkPreviewChance": 0.2,
+        "CustomEmojiChance": 0.2,
```

Comment:
That's a lot of emoji. does that number make sense? What about modeling posts with multiple emoji?

Reply:
It's hard to model accurately because the clients will only pull the emoji once and then never again until the app reloads. So at first they'll be pulling a lot of them, then over time they will pull less. It's a bit on the high end compared to the stats I looked at on pre-release but since it's hard to get an accurate number I'd rather be on the high end.

As for multiple emoji in a post, we could do that but I think it would be just as good to raise the chance of a post having an emoji to be higher